packages/tasks/src/tasks/text-to-3d/about.md

```diff
@@ -4,17 +4,13 @@ Text-to-3D models can be used in a wide variety applications that require 3D, su
 
 This task is similar to the [image-to-3d](https://huggingface.co/tasks/image-to-3d) task, but takes text input instead of image input. In practice, this is often equivalent to a combination of [text-to-image](https://huggingface.co/tasks/text-to-image) and [image-to-3d](https://huggingface.co/tasks/image-to-3d).
 
-### Meshes
+### Generating Meshes
 
 Meshes are the standard representation of 3D in industry.
 
-Meshes can be in `.obj`, `.glb`, `.stl`, or `.gltf` format. Other formats are allowed, but won't be rendered in the gradio [Model3D](https://www.gradio.app/docs/gradio/model3d) component.
+### Generating Gaussian Splats
 
-### Splats
-
-[Gaussian Splatting](https://huggingface.co/blog/gaussian-splatting) is a differentiable rasterization technique that represents 3D as fuzzy points.
-
-Splats can be in `.ply` or `.splat` format. They can be rendered in the gradio [Model3D](https://www.gradio.app/docs/gradio/model3d) component using the [gsplat.js](https://github.com/huggingface/gsplat.js) library.
```

Comment:
I think this can be somewhere way below but I'd still keep it really! (for both pages)